php/exceptions/LorisNotModuleException.class.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * This class is an Exception extension created to distinguish any
+ * Exception from the case when a module can't be loaded because it is an instrument
```

Comment:
This doesn't get thrown because it's an instrument, it gets thrown because the module doesn't exist. The module not existing is regardless of whether or not there's an instrument that exists.